spec/lib/generators/forem/dummy/templates/config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
I added it because it was outputting warnings about missing config (and the fact that simple_form will use its default) when running the specs.